packages/compose/src/hooks/use-container-match/index.native.js

```diff
@@ -0,0 +1,38 @@
+/**
+ * WordPress dependencies
+ */
+import { useState, useCallback } from '@wordpress/element';
+/**
+ * External dependencies
+ */
+import { mapValues } from 'lodash';
+
+/**
+ * Internal dependencies
+ */
+import compareWidths from './compare-widths';
+
+const useContainerMatch = ( queries ) => {
```

Comment:
Can we implement unit-tests by mocking `onLayout ` ?

Reply:
Sure!

Reply:
Added!